pages/probes/[id].vue

```diff
@@ -0,0 +1,367 @@
+				</div>
+				<div>Type:<span class="ml-2 font-bold">{{ probe.hardwareDevice || 'Container' }}</span></div>
+				<div>Version:<span class="ml-2 font-bold">{{ probe.version }}</span></div>
+			</div>
+		</div>
+		<div class="px-5 py-7 dark:text-surface-0">
+			<label for="probeName" class="text-xs">Probe name</label>
+			<InputText
+				id="probeName"
+				v-model="probe.name"
+				class="mt-1 w-full"
+			/>
+			<label for="primary-ip" class="mt-3 inline-block text-xs">Primary IP</label>
+			<div class="relative mt-1">
+				<i class="pi pi-lock absolute right-3 top-2.5 text-bluegray-500"/>
+				<InputText
+					id="primary-ip"
+					v-model="probe.ip"
+					disabled
+					class="w-full bg-transparent dark:bg-transparent"
+				/>
+			</div>
+			<label for="alternative-ips" class="mt-3 inline-block text-xs">Alternative IPs</label>
+			<div class="relative mt-1">
+				<i class="pi pi-lock absolute right-3 top-[13px] text-bluegray-500"/>
+				<AutoComplete
+					id="alternative-ips"
+					v-model="probe.altIps"
+					class="bg-transparent dark:bg-transparent"
+					chip-icon="hidden"
+					multiple
+					disabled
+					:typeahead="false"
+				/>
+			</div>
+			<label for="city" class="mt-3 inline-block text-xs">Location</label>
+			<InputGroup class="mt-1">
+				<InputGroupAddon class="!bg-transparent">
+					<CountryFlag :country="probe.country" size="small"/>
+				</InputGroupAddon>
+				<InputText
+					id="city"
+					v-model="probe.city"
+					class="w-full"
+				/>
+			</InputGroup>
+			<p class="mt-2 text-xs text-bluegray-400">
+				City where the probe is located. If you know that city is wrong it can be changed here: type in the valid city and click save.
+			</p>
+			<label for="tags" class="mt-3 inline-block text-xs">Tags</label>
+			<div v-if="isEditingTags" class="mt-1">
+				<div>
+					<div v-for="(tag, index) in tagsToEdit" :key="index" class="mb-2 flex items-center" :class="{ 'mb-5': !isTagValid(tag.value) }">
+						<Select v-model="tag.uPrefix" class="flex-1" :options="uPrefixes" :scroll-height="'200px'"/>
+						<span class="mx-2">-</span>
+						<span class="relative flex-1">
+							<InputText v-model="tag.value" :invalid="!isTagValid(tag.value)" class="w-full"/>
+							<p v-if="!isTagValid(tag.value)" class="absolute pl-1 text-red-500">Invalid tag</p>
+						</span>
+						<Button icon="pi pi-trash" text aria-label="Remove" class="text-surface-900 dark:text-surface-0" @click="removeTag(index)"/>
+					</div>
+				</div>
+				<div class="mt-1 flex">
+					<Button
+						label="Add tag"
+						icon="pi pi-plus"
+						severity="secondary"
+						class="dark:!bg-dark-800"
+						outlined
+						@click="addTag"
+					/>
+					<Button
+						label="Save"
+						icon="pi pi-check"
+						severity="secondary"
+						outlined
+						class="ml-auto bg-surface-200"
+						@click="saveTags"
+					/>
+					<Button label="Cancel" severity="secondary" outlined class="ml-1 dark:!bg-dark-800" @click="cancelTags"/>
```

Comment:
Let's remove these and save any changes only by clicking the modal's main Save button. It's confusing that tags get saved while the modal is still open.